app/grandchallenge/cases/serializers.py

```diff
@@ -178,14 +189,24 @@ def validate(self, attrs):
         num_targets = sum(f in attrs for f in self.targets)
         if num_targets > 1:
             raise ValidationError(
-                "Only one of archive, answer or reader study can be set"
+                "Only one of archive, archive item, answer or reader study can be set"
             )
 
-        if "interface" in attrs and "archive" not in attrs:
+        if "interface" in attrs and not (
+            "archive" in attrs or "archive_item" in attrs
+        ):
             raise ValidationError(
-                "An interface can only be defined for archive uploads."
+                "An interface can only be defined for archive "
+                "or archive item uploads."
             )
 
+        if "archive_item" in attrs:
+            if "interface" not in attrs:
```

Comment:
This can be simplified to:

```suggestion
        if "archive_item" in attrs and "interface" not in attrs:
```